manifests/ocs/rook-config-override.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rook-config-override
+  namespace: rook-ceph
+data:
+  config: |
+    [global]
+    public network = 10.19.135.0/24
```

Comment:
presumably this is env specific?

maybe some README additions are needed to cover what needs to be customized for each env?

Reply:
@russellb Yup, Could be specific. I could place dummy values.